flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveTestUtils.java

```diff
@@ -45,6 +47,10 @@ public static HiveCatalog createHiveCatalog(String catalogName) {
 		return new HiveCatalog(catalogName, null, createHiveConf(), null);
 	}
 
+	public static HiveCatalog createHiveCatalog(String name, Map<String, String> properties) {
```

Comment:
minor: can we combine this util and the above one? We may have a bit too many utils to create a HiveCatalog :) BTW, I think it can just take a hive version and let callers decide and get whichever version it wants to use